configuration.nix

```diff
@@ -32,8 +32,8 @@
     };
 
     nixPath = [
-      "nixpkgs=channel:nixpkgs-unstable"
-      "darwin=https://github.com/LnL7/nix-darwin/archive/master.tar.gz"
+      "nixpkgs=${inputs.nixpkgs}"
+      "darwin=${inputs.darwin}"
```

Comment:
I wonder if we can outright delete them if we're going to use flakes to build the darwin config overall?

Reply:
at any rate we probably don't want to pin nixpkgs at this level, so the buildkite environment is a bit more "floating."